--- a/chrome/browser/platform_util.cc
+++ b/chrome/browser/platform_util.cc
@@ -70,6 +70,7 @@
 
 }  // namespace internal
 
+#if 0
 void OpenItem(Profile* profile,
               const base::FilePath& full_path,
               OpenItemType item_type,
@@ -87,6 +88,7 @@
       base::BindOnce(&VerifyAndOpenItemOnBlockingThread, full_path, item_type,
                      std::move(callback)));
 }
+#endif
 
 bool IsBrowserLockedFullscreen(const Browser* browser) {
 #if BUILDFLAG(IS_CHROMEOS_LACROS)
