--- a/content/public/common/url_constants.cc
+++ b/content/public/common/url_constants.cc
@@ -14,6 +14,7 @@
 const char kChromeDevToolsScheme[] = "devtools";
 const char kChromeErrorScheme[] = "chrome-error";
 const char kChromeUIScheme[] = "chrome";
+const char kChromeSearchScheme[] = "chrome-search";
 const char kChromeUIUntrustedScheme[] = "chrome-untrusted";
 const char kGuestScheme[] = "chrome-guest";
 const char kViewSourceScheme[] = "view-source";
